packages/pn-commons/src/utils/date.utility.ts

```diff
@@ -96,3 +96,10 @@ export function formatFromString(date: string): Date | null {
   }
   return null;
 }
+
+export function formatDateString(dateString: string): string {
```

Comment:
we can remove this function and update the formatDate. define a second parameter "todayLabelizzation" that is set to true by default. change the condition `isToday(date) ? todayLabel : `${day}/${month}/${date.getFullYear()}`;` in `isToday(date) && todayLabelizzation ? todayLabel : `${day}/${month}/${date.getFullYear()}`;`